l10n_it_fatturapa_pec/tests/test_e_invoice_response.py

```diff
@@ -100,6 +101,30 @@ def test_process_response_INVIO(self):
         self.assertEqual(e_invoices.xml_supplier_id.vat,
                          'IT02652600210')
 
+    def test_process_response_INVIO_broken_XML(self):
+        """Receiving a 'Invio File' with a broken XML sends an email
+        to e_inv_notify_partner_ids"""
+        incoming_mail = self._get_file(
+            'POSTA CERTIFICATA: Invio File 7339338 (broken XML).txt')
+        outbound_mail_model = self.env['mail.mail']
+        error_mail_domain = [
+            ('body_html', 'like', 'unbound_prefix'),
+            ('recipient_ids', 'in',
+             self.PEC_server.e_inv_notify_partner_ids.ids)]
+        error_mails_nbr = outbound_mail_model.search_count(error_mail_domain)
+        self.assertFalse(error_mails_nbr)
+
+        with mock.patch('odoo.addons.fetchmail.models.fetchmail.POP3') \
```

Comment:
Come mai POP invece di IMAP?

Reply:
Mi è risultato leggermente più semplice fare il `mock` perché pop restituisce il numero di messaggi in `num_messages` mentre imap itera su `data[0].split()`.